src/Header/Nav/Nav.module.scss

```diff
@@ -54,7 +54,8 @@
   }
 }
 
-.link {
+a.link {
+  //qualified this with a tag
```

Comment:
This comment would be useful to explain why - ie to add specificity to avoid overrides from application css